src/plugins/telemetry/schema/oss_plugins.json

```diff
@@ -154,6 +154,19 @@
         }
       }
     },
+    "search-session": {
```

Comment:
@elastic/telemetry this config was moved from x-pack to oss. Is it safe to do? Is there anything else to be aware of?

Reply:
I'll let @Bamieh or @afharo confirm, but I think it's fine

Reply:
yup! all good! I don't think it changes the resulting output :)